flink-runtime-web/web-dashboard/app/scripts/modules/submit/submit.ctrl.coffee

```diff
@@ -167,7 +167,7 @@ angular.module('flinkApp')
             $scope.uploader['success'] = null
           else
             $scope.uploader['success'] = "Uploaded!"
-      xhr.open("POST", "/jars/upload")
+      xhr.open("POST", "jars/upload")
```

Comment:
Primarily, this makes all server side requests consistent. Further, if you check in index.coffee, the jobserver URL is set to empty for production, and a local host string for development purposes. This helps with streamlining dashboard dev without having to rebuild the maven modules again and again. 
This particular case was disabling the dev on submit page. Best to get this in while you're fixing all urls.